queue-scm-1.rockspec

```diff
@@ -27,7 +27,8 @@ build = {
         ['queue.abstract.driver.limfifottl'] = 'queue/abstract/driver/limfifottl.lua',
         ['queue.compat']                     = 'queue/compat.lua',
         ['queue.util']                       = 'queue/util.lua',
-        ['queue']                            = 'queue/init.lua'
+        ['queue']                            = 'queue/init.lua',
+        ['queue-version']                    = 'queue-version.lua'
```

Comment:
Let's use `queue/version.lua`.

For modules like `checks`, I also suggest to create a new folder `checks` and a file named `checks/version.lua`.

Reply:
Ok, but we already have `vshard/version.lua` in vshard [repo](https://github.com/tarantool/vshard/tree/master/vshard). Then for `vshard` the name or location of version file will be different 

> Let's use queue/version.lua

ok, I thought this file would be more noticeable in the root of the repository and it would be harder to lose it